### ISSUE TITLE
[Accessibility]: Minimal Form Flow - Statement of truth heading does not follow logical order

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#5946]
### Point of contact

jeana-adhoc

### How severe do you think this issue is?

High

### Severity rationale

This is not blocking a use from completing their task, but it does cause for a confusing experience. 

WCAG 1.3.1

### VA 508 Office

N/A

### URL where issue occurs

https://staging.va.gov/supporting-forms-for-claims/submit-statement-form-21-4138/review-and-submit

### Component or pattern (if applicable)

va-statement-of-truth

### Environment details

Any browser, device. Not Screen reader dependent

### Steps to reproduce

1. Navigate to the review page of any form using the minimal form flow (https://staging.va.gov/supporting-forms-for-claims/submit-statement-form-21-4138/review-and-submit)
2. Observe the heading level of the statement of truth

### Observed behavior

The `Statement of Truth` heading is a child of the last accordion above it, it is an `<h3>`

### Expected behavior / Acceptance criteria

The `Statement of Truth` should be an `<h2>`

### Supporting evidence (screenshots, recordings, code snippets)

Here is a screenshot:
<img width="1021" height="310" alt="Image" src="https://github.com/user-attachments/assets/4f5b3ad2-95a3-4363-91d7-c0403b2791a3" />

### Related issues or systemic notes

This work is currently blocked by #5945 

### Type of issue

- [ ] Axe-core
- [ ] Cognitive | user flows & navigating
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [ ] Focus
- [x] Headings
- [ ] Keyboard
- [ ] Link and button labels
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Screenreader | assistive tech & device support
- [ ] Semantic HTML
- [ ] URL standards
- [ ] Zoom or Magnification
- [ ] Something else

**Comment by tbaker1026** [comment]
Blocked - waiting on design system component update

**Comment by tbaker1026** [comment]
Minimal form - needs to be an H2
